### PR TITLE
Small Excel report output changes

### PR DESCRIPTION
**Pull request by Meghansaha**
**What changes are proposed in this pull request?**

Small changes to the Excel workbook output that `affirm_report_excel` produces:

_**individual sheets:**_

- [x] organize tabs for individual MRs in alphabetical order
- [x] remove Notes column, add two columns for Status and Comment
- [x] show variable labels in bold black instead of gray

_**summary sheet:**_

- [x] make column A "assigned to"
- [x] Due to how the utility functions are utilizing the summary dataframe to create the individual sheets, I have also removed the Notes column and replaced it with Status and Comment in the summary sheet as well.

**If there is an GitHub issue associated with this pull request, please provide link.**
#40

--------------------------------------------------------------------------------

Reviewer Checklist (if item does not apply, mark is as complete)

- [x] Ensure all package dependencies are installed by running `renv::install()`
- [x] PR branch has pulled the most recent updates from master branch. Ensure the pull request branch and your local version match and both have the latest updates from the master branch.
- [-] If a new function was added, function included in `_pkgdown.yml`
- [x] If a bug was fixed, a unit test was added for the bug check
- [x] Run `pkgdown::build_site()`. Check the R console for errors, and review the rendered website.
- [x] Code coverage is suitable for any new functions/features. Review coverage with `withr::with_envvar(new = c("NOT_CRAN" = "true"), covr::report())`. Begin in a fresh R session without any packages loaded.
- [x] R CMD Check runs without errors, warnings, and notes
- [x] `usethis::use_spell_check()` runs with no spelling errors in documentation

When the branch is ready to be merged into master:
- [x] Update `NEWS.md` with the changes from this pull request under the heading "`# affirm (development version)`". If there is an issue associated with the pull request, reference it in parentheses at the end update (see `NEWS.md` for examples).
- [-] Increment the version number using `usethis::use_version(which = "dev")`
- [x] Run `usethis::use_spell_check()` again
- [x] Approve Pull Request
- [x] Merge the PR. Please use "Squash and merge".

### CODE REVIEW

**Comment by Meghansaha** [comment]
@shannonpileggi, can't tag you for review so tagging you here.

**Review by shannonpileggi — APPROVED** [review]
looks great, thank you!

**Comment by Meghansaha** [comment]
@shannonpileggi, I don't have write access so can't merge it myself.

**Pull request merged**
